source/integrate/microsoft-teams-interoperability.rst

```diff
@@ -42,7 +42,7 @@ Set up an OAuth application in Azure
  - **Name**: ``Mattermost MS Teams``
  - **Supported account types**: ``Default value (Single tenant)``
  - **Platform**: ``Web``
- - **Redirect URI**: ``https://(MM_SITE_URL)/plugins/com.mattermost.msteams/oauth-redirect``
+ - **Redirect URI**: ``https://(MM_SITE_URL)/plugins/com.mattermost.msteams-sync/oauth-redirect``
```

Comment:
I ran through the settings and realized the mistake here -- since the plugin id hasn't changed, this path was incorrect.